assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,62 @@
 console.log('***** Music Collection *****')
+
+//create a variable collection that starts an empty array
+
+let collection = [];
+
+
+function addToCollection(title, artist, yearPublished) {
+// Create a new object having the above properties
+    let album = {
+        title: title,
+        artist: artist,
+        yearPublished: yearPublished,
+    }; 
+
+// Add the new object to the end of the collection array
+// Return the newly created object
```

Comment:
Typically you would indent comments to match the line of code they are commenting.